recipes-kernel/linux/linux-mainline_6.2.bb

```diff
@@ -1,10 +1,10 @@
 require recipes-kernel/linux/linux-mainline-common.inc
 
-LINUX_VERSION ?= "6.2+"
+LINUX_VERSION ?= "5.13.5"
```

Comment:
This looks wrong, it's the 6.2 kernel

Reply:
Thanks for catching that, @alistair23  I’ll update it to ensure it reflects the correct 6.2 kernel version.